### PR TITLE
Rework community page

### PR DESCRIPTION
**Pull request by davidak**
Here is my rework of the community page.

I mostly used the existing content, removed unrelated information and created a new layout.

It's not perfect, but better than before.

Feel free to suggest better wording.

### CODE REVIEW

File: teams/nixcon.tt [hunk 1]
```diff
@@ -7,6 +7,10 @@
   from any other OS/FOSS conference.
 </div>
 
+<div class="alert alert-info" style="display:inline-block; padding: 8px 14px;">
```
Comment (garbas) [inline teams/nixcon.tt:10]:
```suggestion
<div class="alert alert-info">
```

I would leave this as default style of bootstrap as possible. You might want to add the title to the alert box, eg: "Help needed!"

File: nixos/community.tt [hunk 2]
```diff
@@ -1,226 +1,98 @@
...
-  <div class="span3 center">
-    <a href="http://www.tudelft.nl/"><img src="[% root %]logo/tudelft.gif" alt="TU Delft logo" /></a>
-  </div>
-  <div class="span3 center">
-    <a href="http://www.uu.nl/"><img src="[% root %]logo/uu.gif" alt="UU logo" /></a>
+  <div class="span4">
+    <h3>Forum</h3>
+    <p>The official forum is the right place to get help from other users and discuss the development of the projects. There are also Announcements, Job offers and Events.</p>
+    <div class="text-center">
+      <a class="btn btn-success"
```
[the remark this answers is not in the document]
Comment (davidak) [reply nixos/community.tt:16]: i don't like the style. it looks broken. especially here:

Screenshot from 2020-04-20 22-09-45

can we change it everywhere to the buttons with normal width?

File: nixos/community.tt [hunk 3]
```diff
@@ -1,226 +1,98 @@
...
+    </ul>
+    <h4>Asia</h4>
+    <ul>
+      <li><a href="https://www.meetup.com/Singapore-NixOS-Nix-User-Group/">Singapore (Singapore)</a></li>
+      <li><a href="http://www.meetup.com/Tokyo-NixOS-Meetup/">Tokyo (Japan)</a></li>
+    </ul>
   </div>
 </div>
 
+<h2>Merch</h2>
```
[the remark this answers is not in the document]
Comment (davidak) [reply nixos/community.tt:92]: ok

File: teams/nixcon.tt (hunk 1 above)
Comment (davidak) [reply teams/nixcon.tt:10]: the default looks broken

Screenshot from 2020-04-20 22-20-21

When i add "Help needed!", there is 2 times "help". any suggestion to better phrase it?

**Comment by davidak** [comment]
@garbas thanks for the feedback!

>Just add a redirect to netlify.toml, since once we merge the change goes public 

i made no change to the structure, just the content. but i can add it in this PR

>Most likely Bugs should be also part of the Contribution guide, somewhere under Learn page.

Yes, it can be a section in a Contribution guide and we can link it from Learn page as "Report Bugs".

>>Documentation (Documentation page)

>You probably ment Learn page

i mean any documentation page that fits. it's mostly links to NixOS manual, so learn might already cover it.

**Comment by davidak** [comment]
>Just add a redirect to netlify.toml, since once we merge the change goes public and it makes no sense to put it in a separate PR.

I `git mv nixos/community.tt community.tt` and committed it to the same commit as the changes, but now it's saved as delete/create instead of rename and the actual changes to the text and page history are lost :(

I wanted to prevent that.

I added it as a second commit, but now i'm unable to edit the first commit.

File: nixos/community.tt [hunk 4]
```diff
@@ -1,225 +1,91 @@
...
-<p>To add yourself to <a href="https://planet.nixos.org/">Planet NixOS</a>,
-fork the <a
-href="https://github.com/NixOS/nixos-org-configurations">nixos-org-configurations</a>
-repo and add the feed source to <a
-href="https://github.com/NixOS/nixos-org-configurations/blob/master/nixos-org/planet-feeds.nix">planet-feeds.nix</a>
-then make a pull request.</p>
+<p class="lead">Nix and NixOS are developed and used by a diverse and welcoming community from all around the world.</p>
 
-<h3>Meetups</h3>
+<p><strong>Check out what the community has made recently in the <a href="https://weekly.nixos.org/">NixOS Weekly</a> newsletter.</strong></p>
```
Comment (davidak) [inline nixos/community.tt]: I wanted to make this more prominently. Now i found a way by making it **bold** :)

File: nixos/community.tt (hunk 2 above)
[the remark this answers is not in the document]
Comment (garbas) [reply nixos/community.tt]: Merch is not there now, so this should not be an issue.

File: teams/nixcon.tt (hunk 1 above)
Comment (garbas) [reply teams/nixcon.tt]: To be honest this change doesn't really fit into this PR. Leave it out and maybe create a new PR and ask NixCon team what they actually want there.

File: nixos/community.tt (hunk 2 above)
[the remark this answers is not in the document]
Comment (davidak) [reply nixos/community.tt]: it will have the same issue on the donate page. what do you think about it?

File: teams/nixcon.tt (hunk 1 above)
Comment (davidak) [reply teams/nixcon.tt]: yeah, right

File: nixos/community.tt (hunk 2 above)
[the remark this answers is not in the document]
Comment (garbas) [reply nixos/community.tt]: One PR at the time. We can also get similar layout there

[the remark this answers is not in the document]
Comment (davidak) [reply nixos/community.tt]: See you there :)

https://github.com/NixOS/nixos-homepage/pull/399#issuecomment-616838176

File: nixos/community.tt [hunk 5]
```diff
@@ -1,226 +1,98 @@
...
+    </div>
   </div>
-  <div class="span3 center">
-    <a href="https://www.3tu.nl/"><img src="[% root %]logo/3tu.jpg" alt="3TU logo" /></a>
+  <div class="span4">
+    <h3>Chat</h3>
+    <p>Another place to get in contact with other users is the official IRC channel:</p>
+    <p><a href="irc://irc.freenode.net/#nixos">#nixos</a> on the <a href="https://freenode.net/">freenode network</a> (<a href="https://logs.nix.samueldr.com/nixos/">logs</a> are available)</p>
+    <div class="text-center">
+      <a class="btn btn-success"
```
[the remark this answers is not in the document]
Comment (davidak) [reply nixos/community.tt]: i don't like the style, but that's unrelated to the content of this page

so i will revert it and we can discuss it in another PR

**Comment by rkoe** [comment]
I *really* dislike this merge. Nothing against a rework in general and nothing against your rework, davidak.

BUT: The merge was too early. Removing useful informations from the homepage is a bad idea. Now, there isn't a link to the bugtracker *anywhere* on the homepage, and effectively also no link to the github-repository anywhere! **First**, a new place should be found on the homepage for this information, and **afterwards** it can be removed from the current location. Not the other way round!

**Pull request merged**